### DISCUSSION TITLE
사용자별 config파일 설정시 일부 어플리케이션 미적용

### DISCUSSION THREAD

**Discussion by JaeungJung** [Riey/kime#581] [Q&A]
먼저 좋은 한글 입력기를 만들어주셔서 진심으로 감사드립니다.

사용자별 환경설정을 위해서 ~/.config/kime/config.yaml을 설정해서 사용하고 있을 때
일부 어플리케이션에서는 이 설정파일이 적용되지 않는 현상이 있어 문의드립니다.

사용자 하나는 두벌식, 하나는 세벌식최종(391)을 사용 중이고
terminal이나 기본 notepad 어플리케이션에서는 사용자별로 한글layout 설정이 작동하는데
deb패키지로 설치한 naver-whale에서는 모든 사용자에서 두벌식으로 작동합니다.
해결할 수 있는 방안이 있을지 문의드립니다.

배포판은 debian 11(bullseye) 기반의 Q4OS이고,
kime버전은 2.5.6으로 kime_debian-buster_v2.5.6_amd64.deb으로 설치했습니다.

**Comment by Riey** [comment]
아마 그 프로그램에서 kime이 안돌아가는거 같은데

whale에서 한영전환 누르면 kime 아이콘이 변하는지 확인해 보세요

**Reply by JaeungJung** [reply]
kime는 구동이 됩니다. 한/영 전환도 가능하고, 한글사용은 됩니다.
다만 home 디렉토리에 있는 환경설정만 유저별로 작동을 안하는 것 같습니다.

세벌식으로 설정해 놓은 유저도 naver-whale에서는 두벌식으로 작동합니다.
에디터나 콘솔은 세벌식으로 동작하고요.
